### PR TITLE
(#380) Fix contacts realtime

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: client/app/lib/socket_listener.coffee [hunk 1]
```diff
@@ -1,78 +1,55 @@
...
+
+    constructor: ->
+        super
+        @queue = async.queue @handleModel, 1
+
+    handleModel: (model, next) =>
+        model.fetch
+            success: (fetched) =>
+                if fetched.get('docType') isnt 'event'
+                    console.log('B')
```
Comment (frankrousseau) [inline client/app/lib/socket_listener.coffee]: Can you remove this log?

File: client/app/lib/socket_listener.coffee [hunk 2]
```diff
@@ -1,78 +1,55 @@
...
+                setTimeout next, 50
+
+    process: (event) ->
+        {doctype, operation, id} = event
+        switch operation
+            when 'create'
+                @queue.push new @models[doctype](id: id)
+
+            when 'update'
+                console.log('A')
```
Comment (frankrousseau) [inline client/app/lib/socket_listener.coffee]: Can you remove this log?

File: client/app/lib/socket_listener.coffee [hunk 3]
```diff
@@ -1,78 +1,55 @@
...
+                @queue.push new @models[doctype](id: id)
+
+            when 'delete'
+                for collection in @collections when model = collection.get id
+                    model.trigger 'destroy', model, model.collection, {}
+
+
+    onRemoteCreateOrUpdate: (fetched) ->
+        console.log('C')
```
Comment (frankrousseau) [inline client/app/lib/socket_listener.coffee]: Can you remove this log?

**Comment by aenario** [comment]
Thanks for the review @frankrousseau , i removed console.logs

**Pull request merged**